bedrock/products/templates/products/vpn/platforms/android.html

```diff
@@ -0,0 +1,167 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "products/vpn/includes/macros.html" import vpn_nav_platform_cta, vpn_platform_content_well with context %}
+
+{% extends "products/vpn/base.html" %}
+
+{% set vpn_countries = 30 %}
+{% set vpn_devices = 5 %}
+{% set vpn_servers = 280 %}
+
+{% block page_title_full %}{{ ftl('vpn-android-page-title') }}{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+
+{% block page_desc %}{{ ftl('vpn-android-page-description') }}{% endblock %}
+
+{% block body_id %}mozilla-vpn-platforms-android{% endblock %}
+
+{% set _utm_source = 'www.mozilla.org-vpn-android-page' %}
+{% set _utm_campaign = 'vpn-product-page' %}
+{% set _params = '?utm_source=' ~ _utm_source ~ '&utm_medium=referral&utm_campaign=' ~ _utm_campaign %}
+
+{% block page_css %}
+  {{ css_bundle('mozilla-vpn-platforms') }}
+{% endblock %}
+
+{% block site_header %}
+  {% with
+    custom_nav_cta=vpn_nav_platform_cta(
+      alt_link_text='vpn-platforms-shared-cta',
+    )
+  %}
+    {% include 'includes/protocol/navigation/navigation.html' %}
+  {% endwith %}
+{% endblock %}
+
+{% block sub_navigation %}
+{% include 'products/vpn/platforms/includes/subnav-mobile.html' %}
+{% endblock %}
+
+{% block content %}
+<main>
+  <section class="vpn-hero">
+    <div class="mzp-l-content mzp-t-content-xl">
+      <div class="vpn-hero-container">
+        <div class="vpn-hero-body">
+          <h1 class="vpn-hero-title">{{ ftl('vpn-platforms-shared-title') }}</h1>
+          <h2 class="vpn-hero-subtitle">{{ ftl('vpn-android-hero-headline') }}</h2>
+          <div class="vpn-hero-desc">
+            <p>{{ ftl('vpn-android-page-description') }}</p>
+
+            <a class="vpn-button" href="{{ url('products.vpn.landing') }}">{{ ftl('vpn-platforms-shared-cta') }}</a>
+
+            <div class="vpn-hero-guarantee">
```

Comment:
Should this have the same styling as on the landing page? Purple with the icon:
![image](https://user-images.githubusercontent.com/205591/112055852-94fac000-8b14-11eb-9316-8a055319d814.png)

Or is the plain italics correct for these pages?

This could also be a `p` for a bit more semantic oomph (though that also gets a default bottom margin).

Reply:
This will get fixed in the design up PR today/tomorrow!